builtin/logical/transit/path_keys.go

```diff
@@ -109,6 +109,9 @@ func (b *backend) pathPolicyRead(
 	if p.Derived {
 		resp.Data["kdf_mode"] = p.KDFMode
 		resp.Data["convergent_encryption"] = p.ConvergentEncryption
+		if p.ConvergentEncryption {
```

Comment:
Can we remove this `if` check? I guess it'll always be set considering that we upgrade the entry while getting the policy.